framework/transport/ieee1905_transport/ieee1905_transport.cpp

```diff
@@ -6,94 +6,91 @@
  * See LICENSE file for more details.
  */
 
-#include <mapf/transport/ieee1905_transport.h>
+#include <bcl/beerocks_defines.h>
+#include <mapf/common/config.h>
+
+#include "ieee1905_transport.h"
 
 #include <unistd.h>
 
-namespace mapf {
+namespace beerocks {
+namespace transport {
+
+using broker::BrokerServer;
 
 void Ieee1905Transport::run()
 {
-    MAPF_INFO("starting 1905 transport.");
+    m_broker = std::make_unique<BrokerServer>(TMP_PATH "/" BEEROCKS_BROKER_UDS);
+    LOG_IF(!m_broker, FATAL) << "Failed creating broker server!";
 
-    // init Local Bus interface, subscribe to specific topics
-    local_bus_ = new LocalBusInterface(Context::Instance());
-    local_bus_->Init();
-    if (local_bus_->subscriber().Subscribe<CmduTxMessage>() < 0) {
-        MAPF_ERR("cannot subscribe to local bus.");
-        return;
-    }
-    if (local_bus_->subscriber().Subscribe<InterfaceConfigurationRequestMessage>() < 0) {
-        MAPF_ERR("cannot subscribe to local bus.");
-        return;
-    }
-    local_bus_->Sync();
-    poller_.Add(local_bus_->subscriber());
+    LOG(INFO) << "starting 1905 transport.";
+
+    // Register broker handlers for internal and external messages
+    m_broker->register_external_message_handler(
+        [&](std::unique_ptr<mapf::Message> &msg, BrokerServer &broker) -> bool {
+            LOG(DEBUG) << "Processing external message: " << msg->type();
+            handle_local_bus_pollin_event(msg);
```

Comment:
The name should change - we don't have a local bus anymore, and since the name "local bus" was always problematic (it is NOT A BUS) - better to change the name to something like `handle_local_pollin_event`.

Reply:
Change to `handle_local_broker_pollin_event`